ocs_ci/templates/workloads/fio/config_s3.fio

```diff
@@ -0,0 +1,34 @@
+# Example test for the HTTP engine's S3 support against Amazon AWS.
```

Comment:
I see one problem with placing this fio config file here, which is that this directory `ocs_ci/templates/workloads/fio` contains yaml file which is processed by some ocs-ci code and pushed to command line arguments of fio run, and so existing yaml files are neither complete nor actual fio config file, while `config_s3.fio` is complete and actual fio config file.

Maybe you can address this by adding a README file, where you explain this difference. Nothing that `config_s3.fio` is ini config file of fio, not jinja template nor yaml dump of option for ocs-ci code, while the other files are.

Reply:
I will add a README because I don't know any other better place to keep it in.
I can create some other directory under `ocs_ci/templates` like `ocs_ci/templates/fioconfig/` but it seems worse because then there will be two directories with fio configs under `ocs_ci/templates`.
I can create a new directory in `conf` directory but `conf` directory is currently used for cluster and OCSCI config files which are more related to CI.
But if you think it should be in a new directory then please suggest the path that the files should be placed in.